.ci/build.sh

```diff
@@ -21,3 +21,9 @@ docker build -t eclipse/che-init-plugin-broker:latest -f ${ROOT_DIR}/build/init/
 docker tag eclipse/che-init-plugin-broker:latest eclipse/che-init-plugin-broker:${TAG}
 docker push eclipse/che-init-plugin-broker:latest
 docker push eclipse/che-init-plugin-broker:${TAG}
+
+# Development image
```

Comment:
is this the dev image for the unified or the init broker?

Reply:
For both of them